src/codemagic/tools/app_store_connect/action_groups/bundle_ids_action_group.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import annotations
+
+from abc import ABCMeta
+from typing import List
+from typing import Optional
+from typing import Sequence
+
+from codemagic import cli
+from codemagic.apple.resources import BundleId
+from codemagic.apple.resources import BundleIdPlatform
+from codemagic.apple.resources import Profile
+from codemagic.apple.resources import ProfileState
+from codemagic.apple.resources import ProfileType
+from codemagic.apple.resources import ResourceId
+
+from ..abstract_base_action import AbstractBaseAction
+from ..action_group import AppStoreConnectActionGroup
+from ..arguments import BundleIdArgument
+from ..arguments import CommonArgument
+from ..arguments import ProfileArgument
+
+
+class BundleIdsActionGroup(AbstractBaseAction, metaclass=ABCMeta):
+    @cli.action(
+        "create",
+        BundleIdArgument.BUNDLE_ID_IDENTIFIER,
+        BundleIdArgument.BUNDLE_ID_NAME,
+        BundleIdArgument.PLATFORM,
+        action_group=AppStoreConnectActionGroup.BUNDLE_IDS,
+        deprecation_info=cli.ActionDeprecationInfo("create-bundle-id", "0.49.0"),
+    )
+    def create_bundle_id(
```

Comment:
Previously defined in https://github.com/codemagic-ci-cd/cli-tools/blob/3670fa624997f17cb437bc88198f7bdee3594f35/src/codemagic/tools/app_store_connect.py#L387-L408